flipperdevices/flipperzero-firmware
# Iclass UI

# What's new

- Adds scene to Picopass plugin

Other notes:

 - This was _heavily_ based on the NFC app
 - includes some storage references to set the stage for saving/downgrading to LF HID Prox
 - It is a shit ton of code, but the minimal amount I could do to replicate the existing functionality.
 - As always, let me know of any changes I need to make
 - `./fbt flash_usb --with-updater --extra-int-apps=picopass `

# Verification 

- Launch Picopass plugin
- Select "Read Card"
- Put Flipper near iClass credential
- Hear chime, see success screen, see raw credential (and decoded if 26 bit)

# Checklist (For Reviewer)
